backend/chalice/api_server/app.py

```diff
@@ -59,10 +61,14 @@ def dispatch(*args, **kwargs):
         resource_path = app.current_request.context["resourcePath"].format(**uri_params)
         req_body = app.current_request.raw_body if app.current_request._body is not None else None
 
+        # Must convert the chalice.MultiDict into a list of tuples. Chalice returns chalice.Multidict which is
```

Comment:
Chalice implements it's own Multidict https://github.com/aws/chalice/issues/1158, which is why we need to convert.